include/connection.rb

```diff
@@ -46,6 +46,8 @@ def sock_puts(str)
   def write(str)
     reconnect unless socket_working?
 
+    puts "Http2: Writing headers: #{str}" if @debug
```

Comment:
In theory yes, but no code in the gem is doing anything but outputting headers. Might be missing some implementation for special cases, since that's the case.

Else it might be better to simply rename the title into `Writing data` or alike, to be more generic.